test/core/consensus/babe/babe_test.cpp

```diff
@@ -405,4 +405,5 @@ TEST_F(BabeTest, SlotLeader) {
   ASSERT_OUTCOME_SUCCESS_TRY(babe->processSlot(slot, best_block_info));
 
   testutil::wait(*thread_pool_.io_context());
+  testutil::wait(*thread_pool_.io_context());
```

Comment:
Why is it doubled?

Reply:
Because there are two `post` calls, which are called on `ThreadPool` thread, so unpredictable.
`ThreadPool` must have "test" constructor, which creates 0 threads